src/as2fm/jani_generator/scxml_helpers/scxml_event.py

```diff
@@ -105,6 +105,7 @@ def must_be_skipped_in_jani_conversion(self):
     def is_bt_response_event(self):
         """Check if the event is a behavior tree response event (running, success, failure).
         They may have no sender if the plugin does not implement it."""
+        # TODO: Remove it when deprecated support for running, success, failure BT events is removed
```

Comment:
Because with the new handling we have a single bt_response event per BT node (from here https://github.com/convince-project/AS2FM/pull/58/files#diff-8021c3394c29f60e934c8bf2dff081894800c6ccead20ed8b4550085cfe4ef9cR276), containing the response result as a param. This means that, with the new version, bt_response events are expected to be always sent, and there is no reason for skipping them anymore (that was the main reason for this method to exist)